packages/rollup-core/src/types/data/l1-data-service.ts

```diff
@@ -112,4 +112,11 @@ export interface L1DataService {
    * @throws An error if there is a DB error.
    */
   markQueuedGethSubmissionSubmittedToGeth(queueIndex: number): Promise<void>
+
+  /**
+   * Gets the total count of L1 RollupStateRoots.
```

Comment:
Would def specify finalized in the comment here if you decide not to change the method name